code/core/src/manager/components/sidebar/Search.tsx

```diff
@@ -45,11 +45,17 @@ const options = {
   ],
```

Comment:
@ghengeveld — Are these changes supposed to be part of this PR?

Reply:
Fixed, thank you!